nursery/acquire-debug-privileges.yml

```diff
@@ -0,0 +1,13 @@
+rule:
+  meta:
+    name: acquire debug privileges
+    namespace: host-interaction/process/modify
+    author: william.ballenthin@fireeye.com
+    scope: basic block
+    att&ck:
+      - Privilege Escalation::Access Token Manipulation [T1134]
+    examples:
+      # - unpacked Cl0p ransomware
+  features:
+    - and:
+      - string: "SeDebugPrivilege"

```

Comment:
This rule looks like it would sit alongside of `modify access privileges` nicely.  

I was thinking of ways to tighten the rule down, but (maybe by looking for `LookupPrivilegeValue`) but this is already covered in other rules.  The string `SeDebugPrivilege` shouldn't really appear errantly in many executables without the intent to adjust privs.  So this rule is good to go in my opinion.

Reply:
i would like to put it next to `match: modify access privileges`, though in my experience, there's often a helper function that does the lookup and modify given a string argument. 

unfortunately, right now, we can't match across functions in capa, so catching all this context would be difficult. still, i'll add an `optional: match: modify access privileges` to catch this when feasible.

Reply:
added in 95d6d53